<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Plot</class>
 <widget class="QWidget" name="Tab_Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="push_refresh">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Refresh</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>650</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="gb_trend">
        <property name="title">
         <string>Trend Plot Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_34">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&amp;Y axis label:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>combo_yAxis</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combo_yAxis">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Structure Index</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Generation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy per FU (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Energy (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vickers Hardness (GPa)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PV enthalpy term (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>α</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>β</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>γ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Volume</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Volume per FU</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Formula Units</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_23">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&amp;X axis label:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>combo_xAxis</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combo_xAxis">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Structure Index</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Generation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy per FU (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Energy (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vickers Hardness (GPa)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PV enthalpy term (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>α</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>β</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>γ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Volume</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Volume per FU</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Formula Units</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="cb_showDuplicates">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Show &amp;duplicate structures and supercells</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="cb_showIncompletes">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Show &amp;incomplete structures</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cb_labelPoints">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&amp;Label points</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="combo_labelType">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>SG Number</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SG Symbol</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enthalpy per FU (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Energy (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vickers Hardness (GPa)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PV enthalpy term (eV)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Volume</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Generation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Structure Index</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Structure Tag</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Formula Units</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="cb_showSpecifiedFU">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Show specific FU</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="edit_showSpecifiedFU">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the numbers of formula units separated by commas and hyphens to be searched here. The text box will be updated with the program's interpretation of your input after exiting the box or pressing return.&lt;/p&gt;&lt;p&gt;Examples:&lt;/p&gt;&lt;p&gt;1 , 3&lt;/p&gt;&lt;p&gt;1 - 4&lt;/p&gt;&lt;p&gt;1 - 3, 5, 7 - 9&lt;/p&gt;&lt;p&gt;Note: structures may not be shown for a particular formula unit until the second generation begins.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Toggle options...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="XtalOpt::XtalOptPlot" name="plot">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XtalOpt::XtalOptPlot</class>
   <extends>QFrame</extends>
   <header location="global">xtalopt/ui/xtalopt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo_yAxis</tabstop>
  <tabstop>combo_xAxis</tabstop>
  <tabstop>cb_showDuplicates</tabstop>
  <tabstop>cb_showIncompletes</tabstop>
  <tabstop>cb_labelPoints</tabstop>
  <tabstop>combo_labelType</tabstop>
  <tabstop>cb_showSpecifiedFU</tabstop>
  <tabstop>edit_showSpecifiedFU</tabstop>
  <tabstop>push_refresh</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>983</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_showSpecifiedFU</sender>
   <signal>toggled(bool)</signal>
   <receiver>edit_showSpecifiedFU</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
